.gitignore

```diff
@@ -18,3 +18,5 @@ zig-out/
 /docgen_tmp/
 
 examples/libs/
+
+.devcontainer
```

Comment:
Can you remove this? Whatever it is, it should go in your global `~/.gitignore` file (details on how to add that at the top of this file.)